docs/commands/oras_login.mdx

```diff
@@ -49,6 +49,12 @@ Log in with username and password in an interactive terminal and no TLS check:
 oras login --insecure localhost:5000
 ```
 
+Log in to registry with customized DNS resolution
+
+```bash
+oras login prod.registry.io:80 --resolve prod.registry.io:80:127.0.0.1:5000 --plain-http
```

Comment:
Below command does the same job with simpler usage:
```suggestion
oras login prod.registry.io --resolve prod.registry.io:5000:127.0.0.1 --plain-http
```
Also this file is auto-generated from `oras login -h` and should be modified on its own. `--resolve` is a common flag for all ORAS CLI commands targeting registry developers and it's better to have a blog to introduce it. @amands98 Are you interested in adding the blog?

Reply:
yeah, I can

Reply:
@amands98 Would you mind creating an issue to track this blog item?